### ISSUE TITLE
`/^\s*?\n/v` 

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_lint#1157]
### Lint Name

no-empty-character-class
no-invalid-regexp

### Code Snippet

```ts
/^\s*?\n/v
```

### Expected Result

No lint

### Actual Result

```
empty character class in RegExp is not allowed
Remove or rework the empty character class (`[]`) in the RegExp deno-lint(no-empty-character-class)


Invalid RegExp literal
Rework regular expression to be a valid deno-lint(no-invalid-regexp)
```

### Version

deno 1.33.1 (release, x86_64-unknown-linux-gnu)
v8 11.4.183.1
typescript 5.0.3
